lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1669,8 +1737,9 @@ def configure_disks(self, vm_obj):
             # increment index for next disk search
             disk_index += 1
             # index 7 is reserved to SCSI controller
-            if disk_index == 7:
-                disk_index += 1
+            if 'nvme' not in self.params['hardware']:
```

Comment:
if `nvme` is in the hardware, the disk_index is allowed to be 7?

Reply:
Yeah, It can be 0 to 14.